dog.html

```diff
@@ -0,0 +1,181 @@
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+</head>
+<body>
+    <div class="header">
+        <h1>Seattle's premiere dog adoption agency</h1>  
+
+    </div>
+    <div class="grid-container">
+    <div class="item1">
+        <nav class="site-nav">
+            <a href="index.html">Home</a>
+            <a href="dog.html">/ Dog</a>
+            <a href="blog.html">/ Blog</a>
+        </nav>
+    </div>
+    </div>
+    <div class="container">
+        <h2>Here are all of our Dogs!</h2>
+        </div>
+    <div class="row">
+        <div class="col-md-3">
+            <div class="thumbnail">
+            <img src="images/alex-lacayo-756508-unsplash.jpg" alt="murphy" style="width:40%">
+                <div class="caption">
+                <p>Murphy</p>
+                <p>Cost to Adopt: $123.45</p>
+                <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                <div class="thumbnail"><a href="images/murphy.jpg">Adopt</a></div>
+                </div>
+            </div>
+        </div>
+            <div class="col-md-3">
+            <div class="thumbnail">
+                <img src="images/nikolay-tchaouchev-785309-unsplash.jpg" alt="poppy" style="width:40%">
+                <div class="caption">
+                <p>Poppy</p>
+                <p>Cost to Adopt: $123.45</p>
+                <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                <div class="thumbnail"><a href="images/poppy.jpg">Adopt</a></div>
+                </div>
+            </div>
+        </div>
+        <div class="col-md-3">
+            <div class="thumbnail">
+                <img src="images/marcus-wallis-471438-unsplash.jpg" alt="jack" style="width:40%">
+                <div class="caption">
+                <p>Jack</p>
+                <p>Cost to Adopt: $123.45</p>
+                <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                <div class="thumbnail"><a href="images/jack.jpg">Adopt</a></div>
+                </div>
+            </div>
+        </div> 
+        <div class="col-md-3">
+            <div class="thumbnail">
+                <img src="images/duffy-brook-350225-unsplash.jpg" alt="duffy" style="width:40%">
+                <div class="caption">
+                <p>Duffy</p>
+                <p>Cost to Adopt: $123.45</p>
+                <p>Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+                <div class="thumbnail"><a href="images/duffy.jpg">Adopt</a></div>
```

Comment:
you'll want to next your html inside of the div on a new line. For example
```
<div>
  <a href="#">Hi</a>
</div>
```

Also, these aren't really links, but buttons. I'd also work to make your alt tags a little more descriptive.